kedro/framework/context/context.py

```diff
@@ -1,11 +1,12 @@
 """This module provides context for Kedro project."""
-
 from copy import deepcopy
 from pathlib import Path, PurePosixPath, PureWindowsPath
 from typing import Any, Dict, Optional, Union
 from urllib.parse import urlparse
 from warnings import warn
 
+import attrs
+from attrs import field
```

Comment:
```suggestion
from attrs import field, frozen
```
Super minor nitpicky suggestions, feel free to ignore 😀